si.proto

```diff
@@ -365,6 +365,9 @@ message UpdateNodeInfo {
     // From Draining state to SCHEDULABLE state.
     // If node is not in draining state, error will be thrown
     DRAIN_TO_SCHEDULABLE = 3;
+
+    // Update node total allocatable resources.
+    UPDATE_NODE_CAPACITY = 4;
```

Comment:
Two things:
* the `noop` option is not needed: why would we want to send a message that is ignored and has no response
* we should really replace the `noop` with a generic `update`, we should not need a new message action when we want to update other values later.

I do think the other names are also need a clean up but that might be better for a separate jira (remove NODE in DRAIN_NODE, change DRAIN_TO_SCHEDULABLE to SCHEDULABLE) to make it more consistent.

Reply:
Hi @wilfred-s 

That makes sense. Can we file a cleanup JIRA as a follow-up?
For this one alone, does it seem good?